ccmlib/repository.py

```diff
@@ -158,14 +158,14 @@ def clone_development(git_repo, version, verbose=False, alias=False):
             process = subprocess.Popen(
                 ['git', 'clone', '--mirror', git_repo, local_git_cache],
                 cwd=__get_dir(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            out = log_info(process, logger)
+            out = log_info(process, logger)[0]
```

Comment:
So, it's actually more idiomatic python to write:
`out, _, _ = log_info(process, logger)`,

but otherwise, this looks like the right fix